app/routes/bestuurseenheid/zitting.js

```diff
@@ -1,20 +1,7 @@
 import Route from '@ember/routing/route';
-import moment from 'moment';
 
 export default class BestuurseenheidZittingRoute extends Route {
-  breadCrumb = {};
-
   model(params) {
     return this.store.findRecord('zitting', params.id);
   }
-
-  async afterModel(zitting) {
-    const bestuursorgaan = await zitting.bestuursorgaan;
```

Comment:
Since the awaiting is not done in the model hooks anymore, the page will render render faster. This also means the relationships are only fetched when the breadcrumb accesses it resulting in the following being rendered for a while:

`Zitting van , op 27 april 2021, 12:11.`

Including the relationships (or keep the awaits) should solve that, but I'm not sure if that's wanted?

I think the current version looks glitchy, but I have that feeling for all templates that depend on lazy loaded relationships.

Reply:
I think at the very least we need fastboot to wait until those relationships are fetched so the data is there for scraping. so this should be solved one way or another.

Reply:
Yea I prefer just including the needed data since I don't like that pages reflow all the time when data comes in. But I was under the impression async was prefered to make the page load faster. I didn't think of the scraping though.